databases/week3/Databases_week3_homework/queries.sql

```diff
@@ -0,0 +1,95 @@
+-- MEAL QUERIES 
+
+-- Get all meals
+SELECT * FROM meal;
+
+-- Add a new meal 
+INSERT INTO meal (title, description, location, `when`, max_reservations, price, created_date) 
+VALUES ('fish lasagna', 'lasagna made with salmon', 'vanløse', '2020-04-04 15:15:15', 9, 99, '2019-04-04 15:15:15');
+
+-- Get a meal with any id
+SELECT * FROM meal WHERE id = 2;
+
+-- update a meal with any id 
+UPDATE meal SET price = price + 10, title = 'fish lasagna : TODAYS SPECIAL' WHERE id = 9;
+
+-- Delete a meal
+-- SET SQL_SAFE_UPDATES = 1;
+
+DELETE meal.* FROM meal WHERE id = 1;
```

Comment:
You don't need to specify any columns for a delete query
```suggestion
DELETE FROM meal WHERE id = 1;
```

Reply:
was a mistake. sorry.